lookup.js

```diff
@@ -0,0 +1,83 @@
+/* An AirTable automation script that updates the Manifest Table
+with details provided in the CSV file table. It effectively acts as a lookup
+with some additional validation steps.
+*/
+
+// Logging statements to provide useful feedback in airtable
+console.log('Starting Update');
```

Comment:
Not sure how scoped logs are, but can you add more information into here? For example, create a:

```js
const logger = (statement) => console.log(`AirTable LOOKUP ${new Date().toISOString()}: ${statement}`)
```